src/ripple/rpc/KeypairForSignature.h

```diff
@@ -24,12 +24,14 @@
 #include <ripple/protocol/PublicKey.h>
 #include <ripple/protocol/SecretKey.h>
 #include <ripple/protocol/Seed.h>
+#include <boost/optional.hpp>
+#include <utility>
```

Comment:
Thanks for adding these includes.  FWIW, the compiler would have caught these missing includes if this file were the first include (virtual first, following BeastConfig.h) in KeypairForSignature.cpp.  I know that doesn't follow our alphabetical includes guideline, but it does follow a recommendation from John Lakos (Large Scale C++ Software Design page 111).  Of course, only the non-unity build would show the problem...